mist-common/src/main/java/edu/snu/mist/common/operators/CountWindowOperator.java

```diff
@@ -69,4 +78,12 @@ public void setState(final Map<String, Object> loadedState) {
     super.setState(loadedState);
     count = (long)loadedState.get("count");
   }
+
+  @Override
+  public void processLeftCheckpoint(final MistCheckpointEvent input) {
+    final Map<String, Object> stateMap = checkpointMap.remove(latestTimestampBeforeCheckpoint);
```

Comment:
Why do you remove the state? When does this can happen? This could return `null` if the `Map` is not set before remove.

Reply:
This is was a mistake. I'll fix this.